lib/flow_commerce/flow_api_v0_client.rb

```diff
@@ -23053,7 +23102,7 @@ def patch(&block)
           def do_request(klass)
             Preconditions.assert_class('klass', klass, Class)
 
-            uri = @uri.to_s
+            uri = path.dup
```

Comment:
either works - I chose here to call the `path` method (`path` is exposed via `attr_reader`)